DAQ_System/Source/LSM303DLHC.cpp

```diff
@@ -0,0 +1,74 @@
+#include "mbed.h"
+#include "LSM303DLHC.h"
+#include "stdio.h"
+ 
+#define MAG_ADDRESS  0x3C
+#define ACC_ADDRESS  0x32
+  
+ 
+LSM303DLHC::LSM303DLHC(PinName sda, PinName scl): _device(sda, scl)
+{
```

Comment:
Re-space the initialization list. You can check out the info on it in the style guide's [second example here](https://google.github.io/styleguide/cppguide.html#Constructor_Initializer_Lists)